treecraft/gui/raxml_dialog.py

```
@@ -0,0 +1,1321 @@
+                                           "continue building your tree with this model.")
+                        
+                        # Add the model to the dropdown and select it
+                        self.model_combobox.addItem(model_from_log)
+                        self.model_combobox.setCurrentText(model_from_log)
+                    elif model_file_exists:
+                        # We have a best model file with additional info
+                        model_info = ""
+                        try:
+                            with open(best_model_file, 'r') as f:
+                                model_info = f.read()
+                        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)